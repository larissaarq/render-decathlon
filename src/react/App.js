import React, { Component } from 'react'
import './App.scss'

class App extends Component {
	render() {
		return (
			<article className="hello-world">
				<h1 className="hello-world__title">Hello World!</h1>
			</article>
		)
	}
}

export default App
