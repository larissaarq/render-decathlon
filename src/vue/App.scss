.hello-world {
  &__title {
    color: #000;
    font-size: 20px;
    text-transform: uppercase;
  }
}
