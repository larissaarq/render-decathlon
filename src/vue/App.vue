<template>
  <article class="hello-world">
    <h1 class="hello-world__title">Hello World!</h1>
  </article>
</template>

<script>
import './App.scss'

export default {
  name: "app"
}
</script>
