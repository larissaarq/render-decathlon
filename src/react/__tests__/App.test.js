import React from 'react'
import App from '../App'
import renderer from 'react-test-renderer'

test('Render app component', () => {
  const component = renderer.create(<App />)
  expect(component.toJSON()).toMatchSnapshot()
})
