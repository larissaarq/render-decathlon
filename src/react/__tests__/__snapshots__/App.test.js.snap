// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render app component 1`] = `
<h1>
  Exemplo de Component
</h1>
`;
